test/project_types/script/layers/infrastructure/script_service_test.rb

```diff
@@ -275,6 +275,16 @@
       it 'should have no errors' do
         assert_equal(script_service_response, subject)
       end
+
+      it 'should have no errors when shop domain is formatted' do
+        assert_equal(script_service_response, script_service.enable(
+          api_key: api_key,
+          shop_domain: 'my.shop.com/',
+          configuration: configuration,
+          extension_point_type: extension_point_type,
+          title: title
+        ))
+      end
```

Comment:
I think there are several improvements needed for this test (and similarly in the disable test below)
1. It doesn't belong in this "describe" block, and should have its own "describe" block
2. It should not call the `script_service.enable` as it should just call `subject`
3. It should use `let(:shop_domain) 'my.shop.com/'`
4. It could just mock out the call to `ScriptServiceAPI::query` to inspect the `shop_domain` parameter.